src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/DefaultRazorProjectServiceTest.cs

```diff
@@ -168,10 +169,10 @@ public void UpdateProject_KnownDocuments()
     {
         // Arrange
         var projectManager = TestProjectSnapshotManager.Create(LegacyDispatcher, ErrorReporter);
-        var hostProject = new HostProject("/path/to/project.csproj", RazorConfiguration.Default, "TestRootNamespace");
+        var hostProject = new HostProject("path/to/project.csproj", RazorConfiguration.Default, "TestRootNamespace");
         projectManager.ProjectAdded(hostProject);
-        var document = new HostDocument("/path/to/file.cshtml", "file.cshtml", FileKinds.Legacy);
-        projectManager.DocumentAdded(hostProject, document, Mock.Of<TextLoader>(MockBehavior.Strict));
+        var document = new HostDocument("path/to/file.cshtml", "file.cshtml", FileKinds.Legacy);
```

Comment:
Turns out this test was invalid. By using a leading `/` in the paths, the project was never found in the project manager, which is a no-op, so the test passed. With this change, the test is correctly updating the project, and no-oping for the right reasons (because no documents have changed)

Reply:
good catch